### PR TITLE
Refactor/line item68

### PR DESCRIPTION
**Pull request by ocshawn**
## Description
Number of 508 fixes from jira: [PEAR-1694](https://gdc-ctds.atlassian.net/browse/PEAR-1694)

3. https://portal.gdc.cancer.gov/v2/manage_sets 
    a. Fix Create Set button does not have proper aria attributes 
    b. # of Items button does not have role

4. https://portal.gdc.cancer.gov/v2/cart 
    a. refactor HOW TO DOWNLOAD FILES IN MY CART? to use mantine Accordion
    b. FILE COUNTS BY PROJECT remove tab index as its not needed 
    c. add tooltips and labels to JSON and TSV download buttons

## Checklist

- [x] Added proper unit tests
- [x] Left proper TODO messages for any remaining tasks
- [x] Scanned for web accessibility with **aXe**, and mitigated or documented
      flagged issues

### CODE REVIEW

**Comment by amlehman** [comment]
Can you change the PR target branch to be jacob? I see some conflicts with the aria-hidden changes from this [PR](https://github.com/NCI-GDC/gdc-frontend-framework/pull/896) and those changes aren't on develop.

File: packages/portal-proto/src/components/DropdownWithIcon/DropdownWithIcon.tsx [hunk 1]
```diff
@@ -74,9 +74,7 @@ interface DropdownWithIconProps {
 export const DropdownWithIcon = ({
   disableTargetWidth,
   LeftIcon,
-  RightIcon = (
-    <Dropdown size="1.25em" aria-hidden="true" data-testid="dropdown icon" />
-  ),
+  RightIcon = <Dropdown data-testid="dropdown-icon" size="1.25em" />,
```
Comment (amlehman) [inline packages/portal-proto/src/components/DropdownWithIcon/DropdownWithIcon.tsx]: Could you keep the `aria-hidden` here per Cemile's comment here? https://github.com/NCI-GDC/gdc-frontend-framework/pull/896#issuecomment-1879278418

File: packages/portal-proto/src/features/cart/Cart.tsx [hunk 2]
```diff
@@ -68,27 +63,31 @@ const Cart: React.FC = () => {
...
-            className={`bg-cartLighterOrange w-full h-full pl-4 ${
-              isCollapsed ? "flex items-center" : "pt-[7px]"
-            }`}
+          <Accordion.Item
+            value="howTo"
+            className="border border-cartDarkerOrange text-secondary-contrast-lighter bg-cartLighterOrange"
           >
-            <span className="text-sm font-bold uppercase font-heading">
+            <Accordion.Control
+              className="hover:bg-[#FFFFFF50] pl-0 relative"
```
Comment (amlehman) [inline packages/portal-proto/src/features/cart/Cart.tsx:80]: Is this supposed to be #FFFFFF?

Comment (ocshawn) [reply packages/portal-proto/src/features/cart/Cart.tsx:80]: its 50% opacity as a way to lighten it on rollover

File: packages/portal-proto/src/components/DropdownWithIcon/DropdownWithIcon.tsx (hunk 1 above)
Comment (ocshawn) [reply packages/portal-proto/src/components/DropdownWithIcon/DropdownWithIcon.tsx]: yes sorry, was getting it confused with other rules

Comment (ocshawn) [reply packages/portal-proto/src/components/DropdownWithIcon/DropdownWithIcon.tsx]:
```suggestion
  RightIcon = (
    <Dropdown size="1.25em" aria-hidden="true" data-testid="dropdown-icon" />
  ),
```

**Comment by cemile-ctds** [comment]
4.a. Could you please set the border-radius to zero? Additionally, for consistency, apply the same style changes to the warning message in the "manage sets" section (please maintain the existing colors). Thank you!
Screenshot 2024-01-12 at 15 35 04

**Comment by ocshawn** [comment]
@cemile-ctds change made please review again

**Review by amlehman — APPROVED** [review]

**Pull request merged**